scripts/lib/k8s.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+ensure_controller_gen() {
+  go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0
```

Comment:
this is fine, since it's idempotent, but probably would be good to do:

```
if !is_installed "controller-gen"; then

fi
```
block around the `go get` in order to prevent unnecessarily calling it when controller-gen is already installed.

Note the `is_installed` Bash function is here:

https://github.com/aws/aws-service-operator-k8s/blob/mvp/scripts/lib/common.sh#L11